src/ert/analysis/_es_update.py

```diff
@@ -887,24 +910,39 @@ def _write_update_report(path: Path, snapshot: SmootherSnapshot, run_id: str) ->
             fout.write(f"Standard cutoff: {snapshot.std_cutoff}\n")
             fout.write(f"Run id: {run_id}\n")
             fout.write(f"Update step: {update_step_name:<10}\n")
-            fout.write("-" * 150 + "\n")
+            fout.write("-" * 183 + "\n")
             fout.write(
-                "Observed history".rjust(56)
+                "Index".rjust(44)
+                + "Observed history".rjust(43)
                 + "|".rjust(17)
                 + "Simulated data".rjust(32)
                 + "|".rjust(13)
                 + "Status".rjust(12)
                 + "\n"
             )
-            fout.write("-" * 150 + "\n")
+            fout.write("-" * 183 + "\n")
             for nr, step in enumerate(update_step):
+                assert isinstance(step, ObservationAndResponseSnapshot)
                 obs_std = (
                     f"{step.obs_std:.3f}"
                     if step.obs_scaling == 1
                     else f"{step.obs_std * step.obs_scaling:.3f} ({step.obs_std:<.3f} * {step.obs_scaling:.3f})"
                 )
+
+                coord_cpy = {**step.obs_coords}
+
+                if "name" in coord_cpy:
+                    del coord_cpy["name"]
+
+                if "time" in coord_cpy:
+                    assert isinstance(coord_cpy["time"], date)
+                    coord_cpy["time"] = coord_cpy["time"].strftime("%Y-%m-%d")
+
+                obs_coord_str = ", ".join(
+                    f"{k}:{v}" for k, v in coord_cpy.items() if k != "name"
```

Comment:
Is it possible to do this when extracting it in the first place? And just return the completed `index` string instead? Think that makes it easier to read.

Reply:
That would also make it easier to include it in the update report in the gui:
![image](https://github.com/equinor/ert/assets/44577479/9bab9249-b5e7-4b27-8d97-0e2472a347ae)